src/screens/userInput/UserInput.tsx

```diff
@@ -63,7 +63,7 @@ export const UserInput = ({ navigation }) => {
   };
 
   const handleDemoPlayClick = () => {
-    play('simulcastmultiview', 'k9Mwad', false);
+    play('multiview', 'k9Mwad', false);
```

Comment:
@sheiladoherty-dolby - For this ticket assigned to you to update the demo stream credentials - https://jira.dolby.net/jira/browse/DIOS-5437. Not sure, if you already started looking into this react native sample app?
I have updated the demo credentials in this React Native sample app as part of the work and testing I was doing in this app for the new WebRTC framework integration.